addons/mail/static/src/core/web/recipients_input.js

```diff
@@ -239,6 +267,17 @@ export class RecipientsInput extends Component {
         ];
     }
 
+    async updateRecipient(newEmail, recipientPartnerId) {
+        await this.orm.write("res.partner", [recipientPartnerId], { email: newEmail });
+        const allRecipients = this.getAllMailThreadRecipients();
+        allRecipients.some((oldRecipient) => {
+            if (oldRecipient.partner_id === recipientPartnerId) {
+                oldRecipient.email = newEmail;
```

Comment:
Not sure what is stored, but aren't we storing normalized version of emails in recipient.email ? Just be sure that if someone pastes ``"Raoulette" <raoulette@example.com>`` we correctly set raoulette in frontend email as it expects normalized email as far as I remember (backend supports whatever you want, the write is ok).

Reply:
ok, probably worth a note in code

Reply:
newEmail could be emailNormalized actually, to highlight we expect a normlized email